netbox/extras/dashboard/widgets.py

```diff
@@ -381,7 +381,12 @@ def render(self, request):
         if request.user.is_anonymous:
             bookmarks = list()
         else:
-            bookmarks = Bookmark.objects.filter(user=request.user).order_by(self.config['order_by'])
+            if self.config['order_by'] == BookmarkOrderingChoices.ORDERING_NAME_AZ:
+                bookmarks = sorted(Bookmark.objects.filter(user=request.user),
```

Comment:
We should define the base queryset (`Bookmark.objects.filter(user=request.user)`) once, and then effect ordering one way or the other depending on the selected option. This ensures that any future modifications to the base queryset are needed only once.